### ISSUE TITLE
`cargo check --manifest-path='my-crate/Cargo.toml'` fails with `error: the manifest-path must be a path to a Cargo.toml file`

### ISSUE THREAD

**Issue by d4h0** [nushell/nushell#5064]
### Describe the bug

Hi,

Executing `cargo check --manifest-path='my-crate/Cargo.toml'` fails with:

`error: the manifest-path must be a path to a Cargo.toml file`

It works in Bash, and it works in Nushell without the quotes around `my-crate/Cargo.toml`.

### How to reproduce

1. Setup a crate named 'my-crate'
2. cd ..
3. Execute `cargo check --manifest-path='my-crate/Cargo.toml'`
4. Execute the same command without the quotes around `my-crate/Cargo.toml`: `cargo check --manifest-path=my-crate/Cargo.toml`

### Expected behavior

Step 3 doesn't fail with the error `error: the manifest-path must be a path to a Cargo.toml file` (as step 4 does).

### Configuration

| key                | value                               |
| ------------------ | ----------------------------------- |
| version            | 0.60.0                              |
| build_os           | linux-x86_64                        |
| rust_version       | rustc 1.59.0 (9d1b2106e 2022-02-23) |
| rust_channel       | stable-x86_64-unknown-linux-gnu     |
| cargo_version      | cargo 1.59.0 (49d8809dc 2022-02-10) |
| pkg_version        | 0.60.0                              |
| build_time         | 2022-03-28 22:50:20 +02:00          |
| build_rust_channel | release                             |
| features           | default, trash, which, zip          |
| installed_plugins  |                                     |

**Comment by d4h0** [comment]
After I found [this issue](https://github.com/nushell/nushell/issues/4248) and experimented a bit more, I think I know what is going on.

It seems, `foo --bar="baz1 baz2"` needs to be written as `foo "--bar=baz1 baz2"` in Nushell (notice the `"` before the `--`).

The problem is, this is pretty unintuitive (coming from Bash), and it doesn't seem to be documented.

I haven't read the docs in detail, yet. However, while skimming it, I couldn't find any example that demonstrates how to pass arguments to external programs, that could contain spaces (which is a common task).

The [Coming from Bash](https://www.nushell.sh/book/coming_from_bash.html) page doesn't mention anything about this, for example.

I looked for a FAQ with a section about common pitfalls, but there doesn't seem to be one.

Is this already mentioned somewhere?

That being said, while Nushell still seems to have a few rough edges, I like it already a lot :)
